packages/rum-core/src/boot/preStartRum.ts

```diff
@@ -85,6 +86,7 @@ export function createPreStartStrategy(
 
       // Expose the initial configuration regardless of initialization success.
       cachedInitConfiguration = initConfiguration
+      addTelemetryConfiguration(serializeRumConfiguration(initConfiguration))
```

Comment:
Suggestion: move this line below , after all early retuns, so we don't collect the configuration if the SDK doesn't actually run

Reply:
The telemetry is drained inside startRum so we will collect the telemetry only if the SDK actually run, but it can save a bit of memory :)